src/com/couchbase/tools/tags/TagProcessor.groovy

```diff
@@ -106,15 +114,26 @@ class TagProcessor {
 
                             if (isStart || isEnd) {
                                 boolean include = restoreMode || match
-                                String commentMarker
-                                if (isStart) {
-                                    commentMarker = "/*"
+                                String marker
+                                if (isPython) {
+                                    var leadingWhitespaceLength = line.length() - line.stripLeading().length()
+                                    marker = " ".repeat(leadingWhitespaceLength) + "'''"
```

Comment:
Python doesn't really have block comments - it's multiline strings. They need to have the same indentation as the code around them, hence why I added the leading whitespace here.